### PR TITLE
Flyperson organs are all completely unidentifiable

### PR DESCRIPTION
**Pull request by tralezab**
### species change approved by nemvar!

All flyperson organs have the same, generic name and description, and their icon is picked from a pool of gross, blackened organ sprites.

There's also a new organ they have that just does completely nothing. It is also disguised and looks the same as all the other organs.

## Why It's Good For The Game

Giving flyperson some more differences and uniqueness, I like the idea of the surgeon opening up a flyperson and just being like "what the fuuuuuuck"

## Changelog
:cl:
add: Flyperson biology has been more defined... to be less defined. Good luck, CMO!
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
yeah it sounds like a pain in the ass

**Comment by ATH1909** [comment]
On the one hand, this is funny, but on the other hand, it doesn't really make much sense, since flypeople are just human(oid)s who've been combined with flies; how would that make your lungs look like a brain (or vice versa)?

**Comment by tralezab** [comment]
They're teleporter accidents.

File: code/modules/mob/living/carbon/human/species_types/flypeople.dm [hunk 1]
```diff
@@ -36,3 +42,62 @@
...
+/obj/item/organ/appendix/fly/Initialize()
+	. = ..()
+	icon_state = pick("brain-x-d", "liver-x", "kidneys-x", "stomach-x", "lungs-x", "random_fly_1", "random_fly_2", "random_fly_3", "random_fly_4", "random_fly_5")
+
+/obj/item/organ/appendix/fly/update_icon()
+	return //don't set name or icon thank you
+
+//useless organs we throw in just to fuck with surgeons a bit more
+/obj/item/organ/fly
+	name = "???"
```
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/species_types/flypeople.dm:95]: would be cooler if these were assigned slightly randomized names, you could have an adjective pool of things like "writing," "twitching," "dripping," "soggy," "flaccid," "darkened," "unidentifiable," "crusty," etc then just make the name generated like "[adjective] [adjective] organ"

**Comment by ATH1909** [comment]
> They're teleporter accidents.

Yeah; as I said, they're just human/fly hybrids made by teleporter accidents.

File: code/modules/mob/living/carbon/human/species_types/flypeople.dm (hunk 1 above)
Comment (tralezab) [reply code/modules/mob/living/carbon/human/species_types/flypeople.dm:95]: alright, i'll do it. great suggestion

**Comment by tralezab** [comment]
added random names to the organs

**Comment by fludd12** [comment]
> are we not both people? does my bloody, bloody organ not beat as yours does? do we not eat with the same twitchy, flaccid organ?

- Buzz Buzzdrin, flyperson rights activist, 2537

**Comment by BarrelOfNapkins** [comment]
Why would a flyperson's brain be in their head? Maybe all their organs got spaghetti'd up when they teleported.

**Pull request merged**